src/main/java/com/conveyal/r5/streets/EdgeStore.java

```diff
@@ -1122,4 +1121,13 @@ public void forEachTemporarilyAddedOrDeletedEdge (IntConsumer consumer) {
         }
     }
 
+    public static class DefaultTravelTimeCalculator implements TravelTimeCalculator {
+
+        @Override
+        public float getTravelTimems(Edge edge, int durationSeconds, StreetMode streetMode, ProfileRequest req) {
```

Comment:
At first I thought this was getTravelTimes with a typo.  Maybe rename to getTravelTimeMilliseconds, to avoid confusion and stay consistent with naming elsewhere (e.g. https://github.com/conveyal/r5/blob/dev/src/main/java/com/conveyal/r5/streets/ParkRideRouter.java#L54)?